data/ecosystem/adopters.yaml

```diff
@@ -95,6 +95,11 @@
   reference: /blog/2023/end-user-q-and-a-02/
   referenceTitle: blog post
   contact: ''
+- name: VTEX
+  url: https://www.vtex.com/
+  components: []
+  reference: 'https://colocatedeventseu2023.sched.com/event/1Jo8E/ingesting-65-tb-of-telemetry-data-daily-through-open-telemetry-protocol-and-collectors-gustavo-pantuza-vtex'
+  contact: ''  
```

Comment:
please provide a contact, a github handle (yours?) is preferred, but it can also be an email address, this will not show up on the website, this is for us to follow up if there are any questions, broken links, etc.